frontend/src/admin/components/sidebar/dropdown/dropdownMenu.scss

```diff
@@ -0,0 +1,43 @@
+@import "src/styles/main";
+
+.c-menu {
+  position: relative;
+
+  &__button{
+    &--active {
+      .c-dropdown {
+        opacity: 1;
+        visibility: visible;
+        position: absolute;
+        background-color: $card-color;
+        border-radius: 5px;
+        top: 60px;
+        left: 0;
+        min-width: 150px;
+        box-shadow: 0 1px 8px rgba(0, 0, 0, 0.3);
+        transform: translateY(-20px);
+        transition: opacity 0.4s ease, transform 0.4s ease, visibility 0.4s;
+
+        p {
+          color: $contrast-color;
+          height: 30px;
+          display: flex;
```

Comment:
Why are you displaying a paragraph as flex?

Reply:
I use flex to display the text center vertical

Reply:
Flex can technically be used, but is for composing multiple elements. The same effect can be achieved by using `line-height` or `padding` – with padding being the best option in this scenario.

Reply:
✅ I have centeret it with line-height: 2; and there is 16px padding on the p tag as well